AppCenter/AppCenter/Internals/Util/MSDispatcherUtil.h

```diff
@@ -3,8 +3,17 @@
 
 #import <Foundation/Foundation.h>
 
+#define MS_DISPATCH_SELECTOR(type, object, selectorName, ...)                                                                              \
```

Comment:
Actually we don't really dispatch here as it's not on the queue/threading context should it really be part of this util?

Reply:
the code of it almost does the same thing. So it looks ok to have it together. Probably better naming helps here...

Reply:
I think we can keep this method here. Its essentially dispatch receiver’s message to target.

Reply:
I think It's more about messaging not dispatching.

Reply:
Well you are right, the doc for `NSInvocation` also mention dispatching.